src/contracts/protocol/configuration/RwaATokenManager.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.10;
+
+import {IRwaATokenManager} from 'src/contracts/interfaces/IRwaATokenManager.sol';
+import {IRwaAToken} from 'src/contracts/interfaces/IRwaAToken.sol';
+import {AccessControl} from 'src/contracts/dependencies/openzeppelin/contracts/AccessControl.sol';
+
+/**
+ * @title RwaATokenManager
+ * @author Aave
+ * @notice Implementation of the RWA aToken manager
+ * @dev Registry for RWA aTokens permissions
+ */
+contract RwaATokenManager is AccessControl, IRwaATokenManager {
+  /// @inheritdoc IRwaATokenManager
+  address public immutable override OWNER;
+
+  /// @inheritdoc IRwaATokenManager
+  bytes32 public constant override AUTHORIZED_ATOKEN_TRANSFER_ROLE =
```

Comment:
No need to couple this role with the one of the `ACLManager`. I d rename to smt like `RWA_ATOKEN_ADMIN_ROLE`

Reply:
however `RWA_ATOKEN_ADMIN_ROLE ` does not include `transfer`. I think it should, as other actions might have different roles within the manager, wdyt?

Reply:
hence, helper functions to smt like `grantATokenAdminRole`

Reply:
good point, let's go with `AUTHORIZED_TRANSFER_ROLE`

Reply:
[4436cdb](https://github.com/aave/aave-v3-horizon/pull/2/commits/4436cdb938bdf6baef14bc28faa40e83707402e9)